test/expected/pg_dump.out

```diff
@@ -35,6 +36,33 @@ INSERT INTO "two_Partitions"("timeCustom", device_id, series_0, series_1) VALUES
 \set QUIET on
 \o
 \c :TEST_DBNAME :ROLE_SUPERUSER
+CREATE VIEW worker_counts AS
+WITH
+  tsdb_workers AS (SELECT * FROM pg_stat_activity WHERE application_name LIKE 'TimescaleDB%')
+SELECT count(*) FILTER (WHERE application_name LIKE '%Scheduler') AS scheduler_count,
+       count(*) FILTER (WHERE application_name LIKE '%Launcher') AS launcher_count
+  FROM tsdb_workers WHERE datname = :'TEST_DBNAME';
```

Comment:
Since the launcher is not connected to specific database this will never find the launcher